pmd-core/src/main/java/net/sourceforge/pmd/cpd/CPDConfiguration.java

```diff
@@ -231,6 +245,15 @@ public static CPDRenderer getCPDRendererFromString(String name, String encoding)
         }
     }
 
+    public static CPDReportRenderer getCPDReportRendererFromString(String name, String encoding) {
```

Comment:
I'll make this new method package private and mark the others as internal API - these are just utility methods to convert a name of a renderer to the actual renderer